src/cljs/main/broadfcui/main.cljs

```diff
@@ -253,8 +253,7 @@
      (aset js/window "testJsException"
            (fn [] (js/setTimeout #(throw (js/Error. "You told me to do this.")) 100) nil)))
    :component-will-receive-props
-   (fn []
-     (init-nav-paths))
+   init-nav-paths
```

Comment:
I disagree with this one. I like for React methods to have an explicit `fn` declaration to show directly what's being used.